src/main/java/org/dinosaur/foodbowl/domain/store/dto/StoreResponse.java

```diff
@@ -0,0 +1,62 @@
+package org.dinosaur.foodbowl.domain.store.dto;
+
+import java.math.BigDecimal;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.dinosaur.foodbowl.domain.store.entity.Address;
+import org.dinosaur.foodbowl.domain.store.entity.Store;
+
+@Getter
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+@AllArgsConstructor
```

Comment:
```suggestion
@Getter
@AllArgsConstructor
@NoArgsConstructor(access = AccessLevel.PROTECTED)
```
저는 이 형식을 통일해서 사용하고 있는데 통일성 있게 `PROTECTED`까지 범위를 열어두는건 어떨까요?